sys/file.c

```diff
@@ -68,8 +69,11 @@ static int badfo_close(file_t *f, struct thread *td) {
   return -EBADF;
 }
 
-fileops_t badfileops = {
-  .fo_read = badfo_read,
-  .fo_write = badfo_write,
-  .fo_close = badfo_close,
-};
+static int badfo_getattr(file_t *f, struct thread *td, vattr_t *buf) {
+  return -EBADF;
+}
+
+fileops_t badfileops = {.fo_read = badfo_read,
```

Comment:
I could not find a way to teach `clang-format` to break line after brace if structure definition does not fit single line. Perhaps you know how to fix it.